### PR TITLE
Minor design + content refresh

### PR DESCRIPTION
**Pull request by coliff**
Hey! I ~~was working on changing the build process to use ParcelJS and~~ took the opportunity to refresh the design and content a little bit.

## Highlights:

- Switched from using Open Sans (via Google Fonts) to using native font-stack (Like Airbnb, Medium, [Bootstrap](https://getbootstrap.com/docs/4.6/content/reboot/#native-font-stack) and others do). Improved performance and still looks great.
- Removed background grain PNG image and used a darker grey which I think looks a bit more flat and modern.
- Used [Bootstrap Grid](https://getbootstrap.com/docs/4.6/layout/grid/) for layout, set the max-width to be wider and used responsive column sizes for some sections. All classes not used get stripped so CSS is tiny.
- Added SVG icons for each of the feature sections and rearranged/restructured content.
- Switched the 'Who Uses H5BP?' section to use SVG logos rather than plain text - looks nicer. For now it shows just six logos in a row - but we could potentially increase to 12. (2 rows of 6 on desktop screens, 4 rows of 4 on tablet and 6 rows of 2 on phone screens)
- Changed the 'Many more' link to go to https://h5bp.net rather than the Wiki (which isn't there anymore)
- Removed the 'Tweet' Twitter JavaScript. The button was very rarely used (4 times in the past year) REF: https://twitter.com/search?q=HTML5%20Boilerplate%3A%20The%20rock-solid%20professional%20front-end%20template%20https%3A%2F%2Fhtml5boilerplate.com%2F%20via%20%40h5bp%20&src=typed_query&f=live
- Removed the deprecated Google Analytics code - Universal Analytics is closing down next year... if we want to add Google Analytics we should setup a new GA4 tracking code which has lots of advantages over Universal Analytics.

## Other small changes:

- Removed Twitter Card's title, description, url and image tags - they aren't needed when the OpenGraph meta tags are defined (I may add a note about this to the docs)
  REF: https://developer.twitter.com/en/docs/tweets/optimize-with-cards/guides/getting-started#twitter-cards-and-open-graph

## Issues / Notes

- SVG images are served as separate static assets rather than being inlined. I'd have preferred them to be inlined -maybe can address in a future PR.
- The version number is manually added right now. We should fix this in a future PR.

## Screenshots from Desktop and iPhone 8 (simulator):

h5bp
h5bp_(iPhone 6_7_8)

FEEDBACK WELCOME!

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,199 +1,208 @@
...
+                <p>Google Analytics snippet setup for improved performance and helpful additional event tracking.</p>
+              </div>
+
+          <div class="col">
+            <img src="modernizr.svg" height="56" alt="">
+            <h3>Modernizr</h3>
+            <p>Get the latest minified versions of <a href="https://modernizr.com">Modernizr</a> feature detection
+              library, complete with a custom build configuration.</p>
+          </div>
+          <div class="col">
```
Comment (roblarsen) [inline src/index.html:123]: @coliff We should remove the server configs or re-word it to promote the server configs repo

**Review by roblarsen — CHANGES_REQUESTED** [review]
It looks good. I'll do a second review tomorrow. We need to do something with the server configs.

File: src/index.html (hunk 1 above)
Comment (coliff) [reply src/index.html:123]: > @coliff We should remove the server configs or re-word it to promote the server configs repo

changing from:

> Apache settings to help you deliver excellent site performance and security. We independently maintain [server configs](https://github.com/h5bp/server-configs) for multiple platforms.

to:

> We independently maintain [server configs](https://github.com/h5bp/server-configs) for multiple platforms to help you deliver excellent site performance and security.

**Review by roblarsen — APPROVED** [review]

**Comment by roblarsen** [comment]
Looks good.

**Pull request merged**